### PR TITLE
Change step click and create component

### LINKED ISSUE #71: [frontend][web] allow click in 1, 2, 3 to navigate between screens
https://www.figma.com/file/BfK7HtPsvxZAwZLg1ebz2u/Tracovid?node-id=22%3A196

### PR DESCRIPTION
**Pull request by diogoteix**
Closes #71

### CODE REVIEW

File: Trackovid19-web/src/app/shared/steps/steps.component.ts [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+})
+export class StepsComponent implements OnInit {
+  @Input() steps: Step[];
+
+  constructor() {}
+
+  ngOnInit(): void {}
+
+  public goToPage(url: string) {
+    console.log(url);
```
Comment (fabioscsilva) [inline Trackovid19-web/src/app/shared/steps/steps.component.ts]: Can we delete this?

Comment (diogoteix) [reply Trackovid19-web/src/app/shared/steps/steps.component.ts]: fixed

**Review by fabioscsilva — APPROVED** [review]

**Review by fabioscsilva — APPROVED** [review]

**Review by hainakus — CHANGES_REQUESTED** [review]
change window.href to router navigation.

**Comment by diogoteix** [comment]
> change window.href to router navigation.

fixed

**Review by hainakus — APPROVED** [review]

**Pull request merged**